internal/photoprism/mediafile.go

```diff
@@ -17,6 +22,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/h2non/filetype"
```

Comment:
Note that we've switched to the `github.com/gabriel-vasile/mimetype` library for mime type detection and that the `fs` package includes useful helper functions to work with file types and the filesystem:
https://github.com/photoprism/photoprism/blob/develop/pkg/fs/mime.go